recipe/conda_build_config.yaml

```diff
@@ -216,23 +216,23 @@ boost:
 boost_cpp:
   - 1.66.0
 bzip2:
-  - 1.0
+  - 1
 cairo:
   - 1.14
 curl:
-  - 7.59
+  - 7
```

Comment:
Still think we should be keeping this as `7.59` or similarly constrained for the lower bound.